exasol/python_extension_common/deployment/language_container_builder.py

```diff
@@ -144,19 +153,22 @@ def _set_language_alias(self) -> None:
     def _add_requirements_to_flavor(self, project_directory: str | Path,
                                     requirement_filter: Callable[[str], bool] | None):
         """
-        Create the project's requirements.txt.
+        Adds project's requirements to the requirements.txt file. Creates this file
+        if it doesn't exist.
         """
         assert self._root_path is not None
-        dist_path = self._root_path / "requirements.txt"
         requirements_bytes = subprocess.check_output(["poetry", "export",
-                                                      "--without-hashes", "--without-urls",
-                                                      "--output", f'{dist_path}'],
+                                                      "--without-hashes", "--without-urls"],
                                                      cwd=str(project_directory))
         requirements = requirements_bytes.decode("UTF-8")
         if requirement_filter is not None:
             requirements = "\n".join(filter(requirement_filter, requirements.splitlines()))
-        requirements_file = self.flavor_base / "dependencies" / "requirements.txt"
-        requirements_file.write_text(requirements)
+        # Make sure the content ends with a new line, so that other requirements can be
+        # added at the end of it.
+        if not requirements.endswith('\n'):
+            requirements += '\n'
```

Comment:
:+1: I think the output of subprocess always ends in a newline - but double-stitched holds better (German saying).